fpgaconvnet/models/modules/Module.py

```diff
@@ -241,9 +193,9 @@ def latency(self):
             then the latency of the module is the largest of
             the two.
         """
-        latency_in  = int((self.rows_in() *self.cols_in() *self.channels_in() )/self.rate_in() )
-        latency_out = int((self.rows_out()*self.cols_out()*self.channels_out())/self.rate_out())
-        return max(latency_in,latency_out)
+        latency_in  = self.repeat*int(np.prod(self.input_shape)/self.rate_in())
```

Comment:
Should we change the names here and to the line below to some more representatives ones like we discussed?
@AlexMontgomerie @Yu-Zhewen